src/App.js

```diff
@@ -1,13 +1,57 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
+
+const placeholderTasks = [
+  {
+    task: 'Organize Garage',
+    id: 1528817077286,
+    completed: false
+  },
+  {
+    task: 'Bake Cookies',
+    id: 1528817084358,
+    completed: false
+  }
+];
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor(props) {
+    super(props);
+    this.state = {
+      tasks: placeholderTasks,
+      task: '',
+      id: 0,
+      completed: false,
+    }
+  }
+
+  addTask = e => {
+    e.preventDefault();
+    const newTask = {
+      task: this.state.task,
+      id: this.state.id,
+      completed: false
+    }
+
+    this.setState({
+      tasks: [...this.state.tasks, newTask],
+      task: '',
+    })
+  }
+
```

Comment:
```suggestion
handleChange = e => {
		const { type, name, value } = e.target;
		const val = type === 'number' ? parseFloat(value) : value;
		this.setState({ [name]: val });
	};
```

Reply:
a more verbose handleChange method that also converts any number represented as a string to an actual number.